packages/editor/src/store/selectors.js

```diff
@@ -1796,3 +1796,7 @@ export const getPostTypeLabel = createRegistrySelector(
 export function isPublishSidebarOpened( state ) {
 	return state.publishSidebarActive;
 }
+
+export const __experimentalIsAdminUser = createRegistrySelector(
+	( select ) => () => select( coreStore ).canUser( 'delete', 'settings' )
```

Comment:
We should also take a look at `useResourcePermissions`, like the navigation menu does.
https://github.com/WordPress/gutenberg/blob/9659455eed4343c455760b2f9076fe3a5bdbf2d7/packages/core-data/src/hooks/use-resource-permissions.ts#L112